meerk40t/gui/icons.py

```diff
@@ -1,17 +1,32 @@
 # ----------------------------------------------------------------------
+from glob import glob
```

Comment:
What's with this import. Are you listing the directories with wildcards and I don't see it? I'll assume you wanted `global` and your IDE was too aggressively helpful.

Reply:
Argghh, I hate it when vscode tries to outsmart me, will fix it